### DISCUSSION TITLE
动态页评论背景颜色是否可以更淡一点或也使用半透明效果

### DISCUSSION THREAD

**Discussion by GD-Slime** [ywmoyue/biliuwp-lite#892] [General]
![image](https://github.com/user-attachments/assets/842f8145-827d-4301-98a3-6577426c8622)

这个颜色感觉还是太突兀了，感觉与之前的黑或浅黑比效果不是太好。如果能实现半透明效果自然是最好的！

**Comment by ywmoyue** [comment]
这个颜色我用取色器取的win11下应用处于聚焦状态下的背景色，可能在不支持mica的Win10上效果很差，半透明效果是个好主意
